main.star

```diff
@@ -328,14 +330,27 @@ def run(plan, args={}):
         elif additional_service == "dora":
             plan.print("Launching dora")
             dora_config_template = read_file(static_files.DORA_CONFIG_TEMPLATE_FILEPATH)
-            dora.launch_dora(
+            dora_config = dora.launch_dora(
                 plan,
                 dora_config_template,
                 all_cl_client_contexts,
                 el_cl_data_files_artifact_uuid,
                 network_params.electra_fork_epoch,
             )
             plan.print("Successfully launched dora")
+            SERVICE_URLS["dora"] = "http://{0}:{1}".format(
+                dora_config.ip_address, dora_config.ports["http"].number
+            )
```

Comment:
This is not working as expected.

```
5d87b7d55815:/app$ env|grep NEXT_
NEXT_PUBLIC_EXPLORER_BASE_URL=http://172.16.0.15:8080
NEXT_PUBLIC_NETWORK_NAME=kurtosis-devnet
NEXT_PUBLIC_BEACON_BASE_URL=http://172.16.0.15:8080
```

Dora listens internally at :8080 but exposes 33154:

```
5c5046128288   dora      http: 8080/tcp -> http://127.0.0.1:33154       RUNNING
```

Is it possible to retrieve the external address from dora_config?

Reply:
Hey @PabloCastellano ! Actually this is expected. The dora explorer and the blobscan services are running within the same isolated docker network created by the enclave so the blobscanner will be able to access dora the explorer on port 8080. 

Port 33154 is the port that a service running on your host docker network can use to get to dora's 8080 port!

Reply:
@tedim52 That won't work because the variables expect an external URL. This way links can be clicked in blobscan and opened in dora. If the internal isolated address:port is used, the link will be broken.

As example, check out https://blobscan.com/block/195071 and click on the Slot number. You are being redirected to dora n because `NEXT_PUBLIC_BEACON_BASE_URL` is set `https://dora.dencun-devnet-11.ethpandaops.io/`

Reply:
Ok, so I am going to remove this functionality. Can you share the link to the issue so I can link it from a comment?